ipfx/nwb_reader.py

```diff
@@ -410,6 +499,14 @@ def get_stim_code(self, sweep_name):
 
         return stim_code
 
+    def get_starting_time(self, data_set_name):
```

Comment:
It looks like this function is duplicated twice (once in `class NwbXReader(NwbReader):` and once in `class NwbMiesReader(NwbReader):`). Maybe consider only defining it once in the `NwbReader` class?

Reply:
It's not exactly duplicated, in one case it is an array, in the other a plain element. The function `get_scalar_string` (which should then also be renamed as it is not about strings anymore) can be used to make that distinction.

So the NwbReader class could look like:

```
    def get_starting_time(self, data_set_name):
        with h5py.File(self.nwb_file, 'r') as f:
            sweep_ts = f[self.acquisition_path][data_set_name]
            return get_scalar_string(sweep_ts["starting_time"].value)
```

Reply:
@t-b Neat solution. I have adopted it. Thanks for a careful review and for the ftp info.